app/search/index/page.tsx

```diff
@@ -17,7 +18,7 @@ export default function Search() {
       ]}
       adobeAnalyticsPageName="search"
     >
-      <SearchResults />
+      <SearchResults showFilter={false} isSearchPage={true} data={mockItems} />
```

Comment:
nit: you can also pass the prop as `isSearchPage` and it's the same as `isSearchPage={true}` but this is a stylistic choice and both work.

Reply:
I'll do the same with showFilter